config.toml.manual

```diff
@@ -0,0 +1,774 @@
+# Sample TOML configuration file for building Rust.
```

Comment:
I don't think you meant to commit this. You can remove it with `git rm config.toml.manual` and then `git commit --amend` and `git push --force`.